### ISSUE TITLE
ENUMs conversion failure

### ISSUE THREAD

**Issue by yarelm** [kyleconroy/sqlc#379]
Hi,

I have defined an ENUM in my pg schema:
```
CREATE TYPE some_type AS ENUM ('PC', 'LAPTOP');
```

and the generated code is:
```
type SomeType string

const (
	SomeTypePC      SomeType = "PC"
	SomeTypeLAPTOP  SomeType = "LAPTOP"
)

func (e *SomeType) Scan(src interface{}) error {
	*e = SomeType(src.([]byte))
	return nil
}
```

When trying to read a row with this type, I get the following panic:
`panic: interface conversion: interface {} is string, not []uint8` 

regarding the first line at the Scan() method.
It seems the sql driver passes a string and not a byte slice

Update: after some debugging, seems to be related to the fact that my pg driver is `pgx`

**Comment by kyleconroy** [comment]
@yarelm the only currently supported driver is `lib/pq`. However, it would not be difficult to change the generated enum code to handle both cases.

**Comment by yarelm** [comment]
Thanks. 
Do you mean changed by `sqlc` or by myself?
I can change it by myself, but wouldn't `sqlc` just overwrite that each generation?

**Comment by kyleconroy** [comment]
> Do you mean changed by sqlc or by myself?

I meant changed by sqlc. You're right that changing it yourself wouldn't really work.

**Comment by kyleconroy** [comment]
@yarelm This has been fixed on master. You can get the fix by downloading the latest build. It will be released next week as part of v1.1.0.
